src/main.cpp

```diff
@@ -12,8 +14,15 @@ int main(int argc, char *argv[])
   // reading the input file name from the command line
   std::string inputFileName = argv[1];
 
+  auto start = std::chrono::high_resolution_clock::now();
+  time_t now = time(0);
+  tm *ltm = localtime(&now);

```

Comment:
Can some of these be `const`?

Reply:
Yes, I think these can be `const`. `start`, `finish`, and `elapsedTime` can be `const`. I am not sure if we need to get the current time `now` and print `ltm`. Was there a reason why it was introduced in the code? In my opinion, we don't need it.

Reply:
I find it interesting to see the total time (i.e. the elapsed time). Start and finish time are probably irrelevant.